MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp

```diff
@@ -1261,6 +1300,163 @@ void EnggDiffractionViewQtGUI::setListWidgetBank(int idx) {
   selectBank->setCurrentRow(idx);
 }
 
+void MantidQt::CustomInterfaces::EnggDiffractionViewQtGUI::
+    fittingRunNoChanged() {
+  try {
+    QString focusedFile = m_uiTabFitting.lineEdit_pushButton_run_num->text();
+    std::string strFocusedFile = focusedFile.toStdString();
+    // file name
+    Poco::Path selectedfPath(strFocusedFile);
+    Poco::Path bankDir;
+
+    // handling of vectors
+    m_fitting_runno_dir_vec.clear();
+    std::string strFPath = selectedfPath.toString();
+    std::vector<std::string> splitBaseName = splitFittingDirectory(strFPath);
+
+    if (selectedfPath.isFile() && !splitBaseName.empty()) {
+
+#ifdef __unix__
+      auto home = Poco::Path().home();
+      bankDir.append(home);
+      bankDir.append(selectedfPath.parent().toString());
+      bankDir.append(selectedfPath);
+#else
```

Comment:
This looks good and works well when I enter the run number (Test 2). But when I browse to one of the focused files I get error pop-ups saying that the file cannot be found. I think that these lines are not working well on linux. I'm looking into it.